command/unwrap.go

```diff
@@ -67,6 +67,15 @@ func (c *UnwrapCommand) Run(args []string) int {
 		return PrintRawField(c.Ui, secret, field)
 	}
 
+	// Check if the original was a list response and format as a list if so
+	if secret.Data != nil &&
```

Comment:
For now, this seems to be the only easiest option we have. A unambiguous way to distinguish the List operation from the rest during unwrapping would have been nice.